packages/trader/src/Modules/CFD/Containers/cfd-server-error-dialog.tsx

```diff
@@ -29,16 +41,7 @@ const CFDServerErrorDialog = ({
     );
 };
 
-CFDServerErrorDialog.propTypes = {
-    clearCFDError: PropTypes.func,
-    disableApp: PropTypes.func,
-    enableApp: PropTypes.func,
-    error_message: PropTypes.string,
-    has_cfd_error: PropTypes.bool,
-    is_cfd_success_dialog_enabled: PropTypes.bool,
-};
-
-export default connect(({ ui, modules }) => ({
+export default connect(({ ui, modules }: any) => ({
```

Comment:
i suggest this as we already have the RootStore class:
```suggestion
export default connect(({ ui, modules }: RootStore) => ({
```
the following import line has to be added as well: `import RootStore from '../../../Stores';`